src/renderer/components/error-boundary/error-boundary.tsx

```diff
@@ -32,7 +38,7 @@ export class ErrorBoundary extends React.Component<ErrorBoundaryProps, State> {
 
   back = () => {
     this.setState({ error: undefined, errorInfo: undefined });
-    navigation.goBack();
+    this.props.observableHistory.goBack();
```

Comment:
Heyy, check out [Law of Demeter](https://en.wikipedia.org/wiki/Law_of_Demeter) for some cool insight ;)